### DISCUSSION TITLE
[预览]v2.0.2

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#2526] [分享 / Share]
- 完善一些组件的描述. (PR #2506, PR #2507 by [shenzhiming88](https://github.com/shenzhiming88))
- 修复直播网页全屏时侧栏没隐藏. (#2484)
- 修复新版首页隐藏首页横幅后布局错位.
- 改善 `v1 风格设置面板` 的样式, 适配上一版的组件详情, 并处理一些弹窗溢出的问题.
- 尝试修复 `自定义顶栏` 更新后设置丢失的问题. (#2501)
- 修复关灯模式下倍速调整的提示被遮挡. (#2466)
- 使用原生剪贴板 API, 删除 `GM_setClipboard`. (#2462)
- 因 API 失效, `自定义顶栏` 取消 `使用季节 Logo` 选项. (#2420)
- 修复 `BiliPlus 跳转` 在稍后再看页面链接错误. (#2322)
- 在原版顶栏的消息 iframe 中也执行脚本来支持夜间模式. 使用 `自定义顶栏` 的用户可以把这个 iframe 屏蔽掉, 链接我写在 `自定义顶栏` 的描述里了. (#2494)
- 修复 `强制固定动态侧栏` 没固定原版顶栏的弹窗. (#2493)
- 修复互动视频里 `启用视频截图` 等按钮反复出现 / `播放前查看封面` 暂停时出现. (#2463, #2453)

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.0.2-preview'>[预览]v2.0.2</a>.</em>

**Comment by SekiBetu** [comment]
![image](https://user-images.githubusercontent.com/38283893/141982491-8e2347a7-97f9-4079-8542-72ba12300fbc.png)
是我的问题还是脚本里的版本号没改

**Reply by the1812** [reply]
我 tag 之前打迟了, 现在重传了个版本

**Reply by Jazzwind** [reply]
请问一下这种zip格式的应该怎么添加进Tampermonkey呀？谢谢！

**Comment by SekiBetu** [comment]
> 请问一下这种zip格式的应该怎么添加进Tampermonkey呀？谢谢！

https://github.com/the1812/Bilibili-Evolved/tree/preview
![image](https://user-images.githubusercontent.com/38283893/142725314-e8d97b17-7344-44f8-a217-ff8f1326416b.png)

**Reply by Jazzwind** [reply]
谢谢！
我之前都是在 https://github.com/the1812/Bilibili-Evolved 安装预览版，然后那里的预览版还是1.12.22的。谢谢

**Comment by LINGMOE-X** [comment]
Github两个源预览版还是v1.12.22，不是v2.0.2

**Reply by the1812** [reply]
https://github.com/the1812/Bilibili-Evolved/tree/preview

**Reply by LINGMOE-X** [reply]
Thanks.

**Comment by MarsTLBAD** [comment]
评论加载不出来了，无论是视频底下的还是动态的

**Reply by the1812** [reply]
有报错吗

**Reply by Geeyun-JY3** [reply]
你这样啥信息没有很难帮你排查，建议带上报错、脚本版本号、浏览器版本号等单独开个 issue。

**Reply by MarsTLBAD** [reply]
现在又加载出来没问题了，所以不确定是脚本的问题还是网站的问题

**Reply by Geeyun-JY3** [reply]
这种不能稳定复现 bug 的情况还是挺常见的🌚
